x/tunnel/keeper/keeper_packet.go

```diff
@@ -0,0 +1,237 @@
+package keeper
+
+import (
+	"fmt"
+
+	sdkerrors "cosmossdk.io/errors"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/bandprotocol/chain/v3/pkg/ctxcache"
+	feedstypes "github.com/bandprotocol/chain/v3/x/feeds/types"
+	"github.com/bandprotocol/chain/v3/x/tunnel/types"
+)
+
+// DeductBaseFee deducts the base fee from fee payer's account.
+func (k Keeper) DeductBasePacketFee(ctx sdk.Context, feePayer sdk.AccAddress) error {
+	basePacketFee := k.GetParams(ctx).BasePacketFee
+	if err := k.bankKeeper.SendCoinsFromAccountToModule(ctx, feePayer, types.ModuleName, basePacketFee); err != nil {
+		return err
+	}
+
+	// update total fees
+	totalFees := k.GetTotalFees(ctx)
+	totalFees.TotalPacketFee = totalFees.TotalPacketFee.Add(basePacketFee...)
```

Comment:
Should it be TotalBasePacketFee, or is it intended to be all packet fee?

Reply:
use TotalBasePacketFee